docs/FAQ.md

```diff
@@ -549,3 +549,7 @@ genqlient will instead generate types with the given names.  (You'll need to avo
 ### … my editor/IDE plugin not know about the code genqlient just generated?
 
 If your tools are backed by [gopls](https://github.com/golang/tools/blob/master/gopls/README.md) (which is most of them), they simply don't know it was updated.  In most cases, keeping the generated file (typically `generated.go`) open in the background, and reloading it after each run of `genqlient`, will do the trick.
+
+### … genqlient fail after `go mod tidy`?
+
+If genqlient fails with an error `missing go.sum entry for module providing package`, this is typically because `go mod tidy` removed its dependencies  because they weren't imported by your Go module.  You can read more about this in golang/go#45552; see in particular [this comment](https://github.com/golang/go/issues/45552#issuecomment-819545037).  In short, if you want to be able to `go run` on newer Go you'll need to have a (blank) import of genqlient's entrypoint in a special `tools.go` file somewhere in your module so `go mod tidy` doesn't prune it.
```

Comment:
I wonder if you want to give a specific example: "if genqlient puts its generated files in xxxx, your tools.go file would look like this.". Though maybe that's not so easy.

Reply:
Good idea, done.